sip/sip-000-stacks-improvement-proposal-process.md

```diff
@@ -0,0 +1,111 @@
+# SIP-000 Stacks Improvement Proposal Process
+
+## Preamble 
+
+Title: Stacks Improvement Proposal Process
+Author: Ken Liao <yukanliao@gmail.com>
+Status: Draft
+Type: Process
+Created: 
+License: BSD-2-Clause
+
+## Abstract
+
+Stacks improvement proposals (SIP) are documented design proposals aimed at improving the Stacks blockchain. They should contain concise technical specifications of features or standards and the rationale behind it. SIPs are intended to be the primary mechanism of proposing new features, for collecting of community input on an issue and for documenting design decisions.
+
+## Specification
+
+**SIP Workflow & States**
+
+Draft → Proposed → Final/Active → Obsolete
+|
+|--------------|--------------|
+↓              ↓              ↓                 
+Deferred    Rejected      Withdrawn
+
+**Draft**
+SIP drafts should be submitted as pull requests to the Blockstack core repository. In this state, further changes can be made. Once a member of the core developer committee has reviewed the SIP and determined that there is support in the community, a SIP number will be assigned. You should not self-assign a SIP number. A core developer reviews the draft for quality and ensure guidelines have been followed. 
+
+**Proposed**
+Prior to the start of mining on the Stacks blockchain, for a SIP to progress to the Proposed stage, acceptance from the core developer committee is required. Post-mining, the core developer team will review SIP’s for quality and adherence to guidelines only.
+
+A core developer will move the SIP to the Proposed stage when the following has been satisfied:
+
+
+1. Evidence of sufficient community support
+2. All public criticism and comments have been adequately addressed
+
+**Final/Active**
+A SIP can be moved to the Final/Active status when a working reference implementation has been provided. Additionally, for a soft-fork SIP to progress to Final/Active status, a clear miner majority is required. Soft-fork BIPS themselves may set additional requirements. For a hard-fork SIP to progress to Final/Active status, adoption from the entire Stacks network economy is required. 
```

Comment:
Typo -- SIPs, not BIPs.  Also, I would avoid saying that a hard fork requires the entire Stacks network economy, since it will be basically impossible for this criterion to be met.  Instead, could this SIP say that one hard fork per year will be executed, with the caveat that additional hard forks will be made in order to fix consensus-critical bugs?